test/verify/check-system-info

```diff
@@ -1023,10 +1023,10 @@ password=foobar
         # RHEL 8 has no SHA1 policy, so do not show it.
         b.click("#crypto-policy-button")
         func = b.wait_not_present if m.image.startswith('rhel-8') or m.image.startswith('centos-8') else b.wait_visible
-        func(".pf-v5-c-menu__item-main .pf-v5-c-menu__item-text:contains('Default:sha1')")
+        func(".pf-v5-c-menu__item-main .pf-v5-c-menu__item-text:contains('Default:SHA1')")
```

Comment:
I think this needs to be `DEFAULT:SHA1`

Reply:
Confirmed by [test failure](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18907-20230623-124129-0ddee8b5-fedora-38-other/log.html#175)